### DISCUSSION TITLE
Application first run: LibreOffice: downloading

### DISCUSSION THREAD

**Discussion by grahamperrin** [helloSystem/hello#380] [General]
![image](https://user-images.githubusercontent.com/192271/189573742-eca02e6b-abae-47a4-936f-f26f78a1dc2f.png)

I got no visible download for a very long time, so I closed the application then used [pkg-install(8)](https://www.freebsd.org/cgi/man.cgi?query=pkg-install&sektion=8&manpath=FreeBSD-Ports) to install from _latest_. 

`FreeBSD mowa219-gjp4-hello-070 13.1-RELEASE-p2 FreeBSD 13.1-RELEASE-p2 GENERIC amd64 1301000 1301000`

**Comment by grahamperrin** [comment]
With Inkscape, no problem: 

![image](https://user-images.githubusercontent.com/192271/189574314-e50d3456-9f65-452e-986c-bb088008d830.png)

**Comment by probonopd** [comment]
Moved to https://github.com/helloSystem/Utilities/issues/133
